.travis.yml

```diff
@@ -2,10 +2,12 @@ sudo: false
 
 language: go
 
+env:
+  global:
+  - GO15VENDOREXPERIMENT=1
```

Comment:
We moved from global contrib/ godeps to per project to make splitting them out easier, and because people complained that sub-project godep revs broke other projects (https://github.com/kubernetes/contrib/pull/374)